pyecharts/charts/basic_charts/bar.py

```diff
@@ -6,9 +6,9 @@
 
 class Bar(RectChart):
     """
-    <<< 柱状图/条形图 >>>
+    <<< Bar Chart >>>
 
-    柱状/条形图，通过柱形的高度/条形的宽度来表现数据的大小。
+    Bar chart, which indicates the size of the data by the height/width of the bar.

```

Comment:
Bar chart  presents categorical data with rectangular bars with heights or lengths proportional to the values that they represent

-- from wikipedia :)